.env.example

```diff
@@ -1,2 +1,3 @@
 SERVER_ADDR=dns:localhost:5230
-BOT_TOKEN=your_bot_token
\ No newline at end of file
+BOT_TOKEN=your_bot_token
+ACCESS_TOKEN=userId:access_token
```

Comment:
userId is telegram user id. use bot @userinfobot to get userId.

Reply:
I figured it out, but what I meant is that it might not be obvious for the users. Can we specify telegram username here (like `@myname`)? Or add the actual info how to get this ID to readme or comments somewhere...

Reply:
readme updated.